### ISSUE TITLE
warn when pool capacity is low to add disks

### ISSUE THREAD

**Issue by bdarcus** [rockstor/rockstor-core#905]
Reading that BTRFS gets buggy when pools get full, might be good to offer something similar to what MS does with Storage Spaces: warn a user when capacity is low, and suggest adding discs.

**Comment by schakrava** [comment]
@bdarcus Thanks for opening this issue. The behavior is not super consistent or perhaps not well understood by me to implement this right now. But I do like to do this sooner than later.

**Comment by bdarcus** [comment]
Yes, I was assuming BTRFS may need some more work to make this possible.
But it's definitely a good UX touch.

**Comment by schakrava** [comment]
Based on my recent testing, qgroup stuff is looking better. I haven't tested balance filters and other stuff that could help with this issue. Moving this to a future milestone.

**Comment by phillxnet** [comment]
@MFlyer have you not recently address a related issue to this one via your red triangle warning icon in share table addition recently.
https://github.com/rockstor/rockstor-core/commit/dd062f151b946d3ce27941d2642dd820d0a3d48d
Not sure which pr that was associated with though.
Linking to another related issue:
[New Feature] Storage utilisation email report #1462

**Comment by MFlyer** [comment]
Hi all,
I guess it was share calculation PR

Mirko
